File: tests/data/cpp/failing/method_conflict_with_comments/ExpectedCurrently.cpp
#include<stdio.h>

class MyClass {
    void run(int argc) {
        printf("too few arguments\n");
        exit(1);
    }
<<<<<<< LEFT
    // central backbone of the algorithm
    void run() {
        printf("hello\n");
    }
||||||| BASE
=======
    void run(bool reallyFast) {
        printf("world\n");
    }
>>>>>>> RIGHT
}

File: tests/data/spork/failing/both_modified_both_add_try/ExpectedCurrently.java
class Cls {
    public static void main(String[] args) {
        try {
            System.out.println("Hello");
<<<<<<< LEFT
        } catch (IllegalArgumentException e) {
            System.out.println("Woopsie!");
            System.out.println("My bad!");
        } finally {
||||||| BASE
        }  finally {
=======
        } catch (IllegalArgumentException e) {
            System.out.println("Oopsie!");
            System.out.println("My bad!");
        } finally {
>>>>>>> RIGHT
            System.out.println("Bye bye!");
        }
    }
}
